packages/server/auth/admin.js

```diff
@@ -537,29 +530,45 @@ router.get(
             type: "card",
             title: req.__("API token"),
             contents: [
+              // api token for user
               div(
                 user.api_token
                   ? span({ class: "mr-1" }, req.__("API token for this user: ")) +
                       code(user.api_token)
                   : req.__("No API token issued")
               ),
+              // button for reset or generate api token
               div(
                 { class: "mt-4" },
                 post_btn(
                   `/useradmin/gen-api-token/${user.id}`,
-                    // TBD localization
                   user.api_token ? req.__("Reset") : req.__("Generate"),
                   req.csrfToken()
                 )
               ),
+              // button for remove api token
+              // TBD Sorry. Need help how to hide correctly button Remove Token when api token is undefined.
+              !user.api_token ? text("") :
```

Comment:
you can just do `user.api_token && div(...)` - the html library remove false, null and undefined values

Reply:
I update code according your comments https://github.com/AleksandrSokolov/saltcorn/commit/7c1a59695920d5f5ad0b588a55a9e1492d51712f